.github/workflows/canary-integration-test.yml

```diff
@@ -78,7 +78,12 @@ jobs:
     - name: wait for ceph to be ready
       run: |
         tests/scripts/validate_cluster.sh
-        kubectl -n rook-ceph get pods
+        kubectl -n rook-ceph get pods 
+        kubectl -n rook-ceph get pods --field-selector=status.phase==Running  | sed '/^csi/d' | sed -n '1!p' | awk '{print $1}' > pods.txt
```

Comment:
Why not get the logs for all pods in the rook-ceph namespace? What are we trying to limit?

Reply:
Initially, I tried that but it was failing with message `to specify the name of the containers`.

Reply:
Ah so we're not getting the logs for the csi pods since they have multiple containers. It would be nice to dump the logs for all containers, but I don't recall `kubectl logs` with that option. We likely need specific commands to get the csi logs for the important container(s), but if we can't have a common helper it may not be worth duplicating the commands everywhere for it.